test/LibraryManager.Test/Microsoft.Web.LibraryManager.Test.csproj

```diff
@@ -17,6 +17,14 @@
         <ProjectReference Include="..\..\src\LibraryManager\Microsoft.Web.LibraryManager.csproj" />
         <ProjectReference Include="..\LibraryManager.Mocks\Microsoft.Web.LibraryManager.Mocks.csproj" />
     </ItemGroup>
+    <ItemGroup>
+      <Reference Include="Microsoft.CSS.Core">
+        <HintPath>..\..\..\..\LibraryManager\lib\Microsoft.CSS.Core.dll</HintPath>
+      </Reference>
+      <Reference Include="Microsoft.JSON.Core">
+        <HintPath>..\..\lib\Microsoft.JSON.Core.dll</HintPath>
+      </Reference>
+    </ItemGroup>
```

Comment:
As discussed offline, these are very old. We made some api's internal in JSON and CSS since then. Using this is very risky.